modules/tinymce/src/core/main/ts/dom/DomSerializerImpl.ts

```diff
@@ -106,7 +107,13 @@ const DomSerializerImpl = (settings: DomSerializerSettings, editor: Editor): Dom
     const targetNode = DomSerializerPreProcess.process(editor, node, args);
     const html = getHtmlFromNode(dom, targetNode, args);
     const rootNode = parseHtml(htmlParser, html, args);
-    return args.format === 'tree' ? rootNode : toHtml(editor, settings, schema, rootNode, args);
+    if (args.format === 'tree') {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore need to figure out how to fake the tree format
```

Comment:
Can you please make sure to add TODOs with the epic Jira number in places you've commented out or done this to make sure we don't forget/miss any?

- [ ] Add TODOs with epic Jira numbers for commented out/asserted code

Reply:
@ashfordneil this hasn't been done, as I asked for it to contain the **Epic Jira** numbers, so we can search for a single thing to make sure all the work was done. In addition to that a number of TODO statements have an invalid JIRA number (TINY-8902). If you want the others, then please include both JIRA numbers but I definitely want the Epic in there.

Reply:
Should be good now. Thanks for the catch on the Typos that I'd made with invalid Jira numbers